blueprints/getting-started/README.md

```diff
@@ -0,0 +1,97 @@
+# HashiCorp Vault Add-on for AWS EKS
+
+> Get started with this add-on by reviewing the following example.
+
+## Table of Contents
+
+- [HashiCorp Vault Add-on for AWS EKS](#hashicorp-vault-add-on-for-aws-eks)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+  - [Usage](#usage)
+  - [Variables and Outputs](#variables-and-outputs)
+
+## Overview
+
+The code in this directory showcases an easy way to get started with the HashiCorp Vault Add-on for AWS EKS.
+
+* [main.tf](./main.tf) contains the AWS and Kubernetes resources needed to use this add-on.
+* [outputs.tf](./outputs.tf) defines outputs that make interacting with `kubectl` easier
+* [terraform.tf](./terraform.tf) defines the required Terraform (core) and Terraform provider versions
+* [variables.tf](./variables.tf) defines the variables needed to use this add-on.
+
+## Usage
+
+Initialize the root module and any associated configuration for providers and child modules by executing the `terraform init` command.
+
+Once all dependencies have been installed, execute `terraform plan` and review the resources that will be created.
+
+If you are satisfied with the proposed settings, execute `terraform apply` to create the resources and deploy HashiCorp Vault to an EKS Cluster.
+
+For more detailed information, see the documentation for the [Terraform Core workflow](https://www.terraform.io/intro/core-workflow).
+
+On successful completion, Terraform will display outputs containing URLs to the AWS Console as well as `kubectl`-specific commands.
+
+These commands may be used to configure a local `vault` agent and intialize the server, as described in the next section.
+
+### Unsealing Vault
```

Comment:
*Nit* - It may help to reduce some duplication between READMEs and use either the main README or this one, then provide a link between the two for this section (and any others that make sense).

Reply:
Great catch - discussing this with @im2nguyen now.